site/gr/index.html

```diff
@@ -0,0 +1,672 @@
+                        <li><a href="javadoc/8.1.0">8.1.0</a></li>
+                        <li><a href="javadoc/8.0.0">8.0.0</a></li>
+                        <li><a href="javadoc/7.1.0">7.1.0</a></li>
+                    </ul>
+                </li>
+                <li class="dropdown">
+                    <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true"
+                       aria-expanded="false">Γλώσσα<span class="caret"></span></a>
+                    <ul class="dropdown-menu">
+                        <li><a href="">Αγγλικά</a></li>
+                        <li><a href="de/index.html">Γερμανικά</a></li>
+                        <li><a href="es/index.html">Ισπανικά</a></li>
+                        <li><a href="cn/index.html">Κινέζικα</a></li>
+                        <li><a href="fr/index.html">Γαλλικά</a></li>
+                        <li><a href="ja/index.html">Γιαπωνέζικα</a></li>
+                        <li><a href="pt-br/index.html">Πορτογαλικά-Βραζιλίας</a></li>
+                        <li><a href="ru/index.html">Ρωσικά</a></li>
+                    </ul>
+                </li>
+            </ul>
+        </div>
+        <!-- /.navbar-collapse -->
+    </div>
+    <!-- /.container -->
+</nav>
+
+
+<!-- Header -->
+<a class="anchor" name="about"></a>
+<div class="intro-header">
+    <div class="container-fluid">
+
+        <div class="row">
+            <div class="col-lg-push-2 col-lg-8">
+                <div class="intro-message">
+                    <h1>Eclipse <span style="font-weight: 400">Collections</span></h1>
+                    <h3>- Δυνατότητες που θέλετε με τα collections που χρειάζεστε -</h3>
+                </div>
+            </div>
+        </div>
+
+    </div>
+    <!-- /.container -->
+
+</div>
+<!-- /.intro-header -->
+
+<!-- Page Content -->
+
+<a class="anchor" name="start"></a>
+<div class="content-section-a">
+
+    <div class="container-fluid">
+        <div class="row">
+            <div class="col-lg-5 col-sm-6">
+                <hr class="section-heading-spacer">
+                <div class="clearfix"></div>
+                <h2 class="section-heading">Ξεκινώντας με το <br>Eclipse Collections</h2>
+                <p class="lead">Το Eclipse Collections είναι το καλύτερο Java framework για collections
+                    και σας φέρνει πάντα ευχαρίστηση όταν πρόκειται για ανάπτυξη εφαρμογών Java.
+                    <br><br>Ενσωμαατώστε το σήμερα στο στην εφαρμογή σας μαζί με τα αγαπημένα σας build tools!</p>
+            </div>
+            <div class="col-lg-7 col-sm-6">
+                <ul class="nav nav-tabs">
+                    <li class="active"><a href="#maven" data-toggle="tab">Maven</a></li>
+                    <li><a href="#gradle" data-toggle="tab">Gradle</a></li>
+                    <li><a href="#ivy" data-toggle="tab">Ivy</a></li>
+                    <li><a href="#download" data-toggle="tab">Κατέβασμα</a></li>
```

Comment:
This is not wrong, but I have also seen it as:

```suggestion
                    <li><a href="#download" data-toggle="tab">Λήψη</a></li>
```